client/src/components/main_section/main_section.coffee

```diff
@@ -12,7 +13,7 @@ SingleRepo = React.createFactory require '../repo_list/single_repo.coffee'
 
 MainSection = React.createClass
   render: ->
-    {repo, repoActions, login, pageLocation, pageLocationActions,loginActions} = @props
+    {repo, repoActions, login, pageLocation, pageLocationActions,loginActions, registerActions} = @props
```

Comment:
They would, if you `connect` them as in https://github.com/acmcarther/catalyst/blob/acm-client-registration/client/src/containers/app.coffee . I deff don't want to pass `dispatch` down in props though.